code/workspaces/web-app/src/containers/projects/ProjectsContainer.js

```diff
@@ -21,37 +75,46 @@ class ProjectsContainer extends Component {
   }
 
   adaptProjectsToStacks() {
-    return this.props.projects.value.projectArray.map(project => ({
-      id: project.id,
-      key: project.key,
-      displayName: project.name,
-      description: project.description,
-      accessible: project.accessible,
-      type: 'project',
-      status: 'ready',
-    }));
+    return this.props.projects.value.projectArray.map(projectToStack);
   }
 
   projectUserPermissions(project) {
     return project && project.accessible ? [PROJECT_OPEN_PERMISSION] : [];
   }
 
   render() {
+    const { classes } = this.props;
     return (
       <PromisedContentWrapper promise={this.props.projects}>
         {this.props.projects.value.projectArray ? (
-          <StackCards
-            stacks={this.adaptProjectsToStacks()}
-            typeName={TYPE_NAME}
-            openStack={project => this.props.history.push(`/projects/${project.key}/info`)}
-            deleteStack={() => { }}
-            openCreationForm={() => { }}
-            userPermissions={project => this.projectUserPermissions(project)}
-            createPermission=""
-            openPermission={PROJECT_OPEN_PERMISSION}
-            deletePermission=""
-            editPermission=""
-          />
+          <div>
```

Comment:
I'm in two minds about the CodeClimate warning for this code, it is essentially three lines of code spread over 25 in the name of readability. I'm not a huge fan of the new line per prop as I prefer code to be a bit denser but at the same time don't feel strongly enough to do it differently. Seems silly to break this into functions as it brings little benefit.